install/common-pre-install/prepare_site_init-cpi.md

```diff
@@ -0,0 +1,422 @@
+# Prepare Site Init
+
+These procedures guide administrators through setting up the `site-init` directory which contains important customizations for various products.
+
+1. [Background](#1-background)
+1. [Create and Initialize Site-Init Directory](#2-create-and-initialize-site-init-directory)
+1. [Create Baseline System Customizations](#3-create-baseline-system-customizations)
+1. [Customer-Specific Customizations](#4-customer-specific-customizations)
+
+## 1. Background
+
+The `shasta-cfg` directory included in the CSM release tarball includes relatively static, installation-centric artifacts, such as:
+
+- Cluster-wide network configuration settings required by Helm charts deployed by product stream Loftsman manifests
+- [Sealed Secrets](https://github.com/bitnami-labs/sealed-secrets)
+- Sealed Secret Generate Blocks -- a form of plain-text input that renders to a Sealed Secret
+- Helm chart value overrides that are merged into Loftsman manifests by product stream installers
+
+## 2. Create and initialize Site-Init directory

```

Comment:
This section should be updated with https://github.com/abhishek-g-suresh/docs-csm/blob/main/install/pre-installation.md#13-first-log-in